client/src/components/closedTasksController.js

```diff
@@ -0,0 +1,16 @@
+function recentTasksController($http) {
+  let self = this;
+
+  $http.get('/api/recenttasks').then(function(res) {
+    let tasks = res.data;
+    tasks = tasks.filter((item) => {
```

Comment:
Filtering should be done on a server side. We do not need to send the whole data through the network, when we need only several recent items